File: src/main/java/com/devmpv/repositories/BoardRepository.java
package com.devmpv.repositories;

import org.springframework.data.repository.PagingAndSortingRepository;
import org.springframework.stereotype.Repository;

import com.devmpv.model.Board;

/**
 * Board repository.
 * 
 * @author devmpv
 *
 */
@Repository
public interface BoardRepository extends PagingAndSortingRepository<Board, String> {

}

File: src/main/kotlin/com/devmpv/repositories/BoardRepository.kt
package com.devmpv.repositories

import com.devmpv.model.Board
import org.springframework.data.repository.PagingAndSortingRepository
import org.springframework.stereotype.Repository

/**
 * Board repository.
 *
 * @author devmpv
 */
@Repository
interface BoardRepository : PagingAndSortingRepository<Board, String>
